### PR TITLE
[INLONG-840][Doc] Adding more details to the tubemq-manager documentation

### LINKED ISSUE #840: [Improve][Doc] Adding more details to the tubemq-manager documentation
### Description

Adding more details to the tubemq-manager documentation

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by sunrisefromdark**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #840

### Motivation

**Some parts of the original document are easily misleading to users, it has been modified and supplemented.**

### Modifications

**1、the previous text has been imported sql file, configuration instructions here, can be changed to:
-Add mysql information in conf/application.properties (you can directly use the database imported through sql/apache_tube_manager.sql)

2、Start the manager service according to the configuration above. If you change the applications.properties file in the middle of the process, you need to shut down and restart the manager service for the new configuration to take effect.**

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
Please add the related Chinese document at the same time.
image

File: docs/modules/tubemq/tubemq-manager/quick_start.md [hunk 1]
```diff
@@ -14,14 +14,16 @@ mysql -uDB_USER -pDB_PASSWD < sql/apache_tube_manager.sql
 ````
   
 ## Configuration
-- create `tubemanager` and account in MySQL.
-- Add mysql information in conf/application.properties:
-
 ```ini
 # mysql configuration for manager
```
Comment (dockerzhang) [inline docs/modules/tubemq/tubemq-manager/quick_start.md:18]:
```suggestion
# MySQL configuration for Manager
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/tubemq-manager/quick_start.md [hunk 2]
```diff
@@ -14,14 +14,16 @@ mysql -uDB_USER -pDB_PASSWD < sql/apache_tube_manager.sql
 ```
   
 ## 配置
-- 在mysql中创建`tubemanager`数据和相应用户.
-- 在conf/application.properties中添加mysql信息：
 
 ```ini
 # mysql configuration for manager
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/tubemq-manager/quick_start.md:18]:
```suggestion
# MySQL configuration for Manager
```

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Comment by healchow** [comment]
@sunrisefromdark Please [create an issue first](https://github.com/apache/inlong-website/issues/new/choose), and then change the `[INLONG-834]`  in this PR's title, it should be your new issue number, thanks.

**Pull request merged**